build-dependencies/dependencies/src/main/kotlin/Versions.kt

```diff
@@ -2,7 +2,7 @@ object versions {
     const val coil = "2.1.0"
 
     object jetbrains {
-        const val coroutines = "1.3.7"
+        const val coroutines = "1.6.1"
```

Comment:
Previously coroutines library was resolved automatically through transitive dependency of `io.coil` library to version 1.6.1.